dune

```
@@ -24,4 +24,16 @@
     %{target}
     (run %{ood_gen} planet_feed)))))
 
+(subdir
+ asset/
+ (rule
+  (target changelog_feed.xml)
```

We can call it `changelog.xml`:
```suggestion
  (target changelog.xml)
```